cypress/support/pages/userPage.js

```diff
@@ -36,6 +36,22 @@ export function fillUpUserDetails(firstName, lastName, email) {
     .waitForRequestAndCheckIfNoErrors("@StaffMemberUpdate");
 }
 
+export function fillUpUserDetailsWithNotUniqueEmail(
```

Comment:
**question**: do we have somewhere helper that fill up with unique email?

Reply:
Ill change the name as its misleading, good catch